reana_db/alembic/versions/20210816_0752_6568d7cb6710_full_workflow_workspace.py

```diff
@@ -0,0 +1,60 @@
+"""full workflow workspace.
+
+Revision ID: 6568d7cb6710
+Revises: f84e17bd6b18
+Create Date: 2021-08-16 07:52:03.968797
+
+"""
+
+from sqlalchemy.sql import table, column
+from sqlalchemy import String
+from alembic import op
+from sqlalchemy_utils import UUIDType
+from reana_commons.config import SHARED_VOLUME_PATH
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "6568d7cb6710"
+down_revision = "f84e17bd6b18"
+branch_labels = None
+depends_on = None
+
+
+workflow = table(
+    "workflow",
+    column("workspace_path", String),
+    column("owner_id", String),
+    column("id_", String),
+    schema="__reana",
+)
+
+
+def upgrade():
+    """Upgrade to 6568d7cb6710 revision."""
+    connection = op.get_bind()
+    connection.execute(
+        workflow.update()
+        .where(sa.not_(workflow.c.workspace_path.like("/%")))
+        .values(
+            {"workspace_path": SHARED_VOLUME_PATH + "/" + workflow.c.workspace_path}
```

Comment:
Instead of `foo + "/" + bar`. you may write `os.path.join(foo, bar)` which may be more readable.  (And it would ready to work on some exotic systems where the directory separator and/or joining process may differ!)

Reply:
Hmm, there is some issue there, os.path.join doesn't accept `workflow.c.workspace_path`, as it is from a `ColumnClause` type, not a str or similar. Perhaps I could use `foo+os.path.sep+bar`?

Reply:
Have you tried explicit string cast? Something like: 

```python
os.path.join(SHARED_VOLUME_PATH, str(workflow.c.workspace_path))
```

Reply:
Yes, It stores the name of the column but not the value I need, something like `/var/reana/__reana.workkflow.c.workspace_path`

Reply:
OK, so let's go with the original format then (perhaps with `os.sep` if you prefer, but not necessary...)